app/controllers/ReportOneLoginProblemController.scala

```diff
@@ -0,0 +1,179 @@
+  private val dateOfBirthValidator   = DateOfBirthValidator()
+
+  def form: Form[ReportOneLoginProblemForm] = Form[ReportOneLoginProblemForm](
+    mapping(
+      "name"               -> text
+        .verifying(
+          "problem_report.name.error.required",
+          name => name.nonEmpty
+        )
+        .verifying(
+          "problem_report.name.error.length",
+          name => name.length <= 70
+        )
+        .verifying(
+          "forms.name.error.invalid",
+          name => nameValidator.validate(name) || name.isEmpty
+        ),
+      "nino"               -> text
+        .verifying(
+          "one_login_problem.nino.error",
+          nino => nino.nonEmpty
+        )
+        .verifying(
+          "one_login_problem.nino.error",
+          nino => taxIdentifierValidator.validateNino(nino) || nino.isEmpty
+        ),
+      "sa-utr"             -> optional(
+        text
+          .verifying(
+            "one_login_problem.sa-utr.error",
+            saUtr => taxIdentifierValidator.validateSaUtr(saUtr)
+          )
+      ),
+      "date-of-birth"      -> mapping(
+        "day"   -> text.verifying("one_login_problem.date-of-birth.error.day", day => day.nonEmpty),
+        "month" -> text.verifying("one_login_problem.date-of-birth.error.month", month => month.nonEmpty),
+        "year"  -> text.verifying("one_login_problem.date-of-birth.error.year", year => year.nonEmpty)
+      )(DateOfBirth.apply)(d => Some(Tuple.fromProductTyped(d)))
+        .verifying("one_login_problem.date-of-birth.error.invalid", dob => dateOfBirthValidator.isValidDate(dob))
+        .verifying(
+          "one_login_problem.date-of-birth.error.future",
+          dob => dateOfBirthValidator.isNotFutureDate(dob) || !dateOfBirthValidator.isValidDate(dob)
+        ),
+      "email"              -> text
+        .verifying(
+          s"problem_report.email.error.required",
+          email => email.nonEmpty
+        )
+        .verifying(
+          s"problem_report.email.error.valid",
+          email => emailValidator.validate(email) || email.isEmpty
+        ),
+      "phone-number"       -> optional(text),
+      "address"            -> text
+        .verifying(
+          s"one_login_problem.address.error",
+          address => address.nonEmpty
+        ),
+      "contact-preference" -> of[ContactPreference],
+      "complaint"          -> optional(text),
+      "csrfToken"          -> text
+    )(ReportOneLoginProblemForm.apply)(o => Some(Tuple.fromProductTyped(o)))
+  )
+
+  def emptyForm(csrfToken: String): Form[ReportOneLoginProblemForm] =
+    ReportOneLoginProblemFormBind.form.fill(
+      ReportOneLoginProblemForm(
+        name = "",
+        nino = "",
+        saUtr = None,
+        dateOfBirth = DateOfBirth.empty,
+        email = "",
+        phoneNumber = None,
+        address = "",
+        contactPreference = EmailPreference,
+        complaint = None,
+        csrfToken = csrfToken
+      )
+    )
+}
```

Comment:
if we remove the csrf token from the model then we don't need to have this, we can just use `ReportOneLoginProblemFormBind.form` in the index action, the formWithCsrf component we use in the view will output the csrf token input field for us so no need to prepopulate here

Reply:
@oscarduignan sounds good, see comment above re: tech debt to do this for the other forms too.

Reply:
@oscarduignan done